src/Standards/Squiz/Docs/Commenting/BlockCommentStandard.xml

```diff
@@ -0,0 +1,280 @@
+<documentation title="Block Comment">
+    <standard>
+    <![CDATA[
+    A block comment is a multiline comment delimited by an opener "/*" and a closer "*/" which are each on their own line and the content is in between.
```

Comment:
Just checking: is it "multiline" or "multi-line" ? Whichever it should be, let's use it consistently (here versus line 48 of the docs "Valid: Multi-line block comment").

Also, how about this ?

```suggestion
    A block comment is a multiline comment delimited by an opener "/*" and a closer "*/" which are each on their own line with the comment text in between.
```

Reply:
From what I can see, UK English uses multi-line, US English doesn't.

Which would you prefer?

Reply:
I prefer consistency, whether it is the one or the other ;-)

I know there are some typical words which are written differently in US vs UK English - "modernize" vs "modernise" comes to ming.
Might be worth checking the code base/docs to see what's mostly used, US or UK English, though I'm not sure if this was strictly adhered to in the past anyhow.

If a spot check shows a mix of both, I suggest we open an issue to choose one and make the docs consistently use that language variant (as a separate task, outside the scope of this PR).

Reply:
Checking a small number of differences, such as `multi-line` vs `multiline`, `ise` vs `ize`, `colour` vs `color`, and such shows a mixture in the repository. Some spellings occur more in comments, others in docs, others in code, and others differ across three of those areas. Obviously `color/background-color` in CSS doesn't count, but for example, there's `NamedColourSniff` and `T_COLOUR`, but also the `private colorizeVariableInput()` method in the Help utility class.

Regarding this PR, it looks like `multi-line` is used more often outside of things like PHP class names where the hyphen isn't an option. I'll update the PR to use `multi-line` consistently. At least it'll make it easier to change in one go should the variant of English be changed in the future.

Reply:
Thanks for checking that @costdev. I'm not too concerned about the CSS sniff/tokens as those will be removed soon anyway, but I think opening an issue about language consistency might be a good idea. Want to do the honours ?